adis/adis.py

```diff
@@ -158,3 +166,22 @@ def dumps(self):
 
     def __repr__(self):
         return """Adis containing %d Adis-files""" % len(self.files)
+
+    @staticmethod
+    def add_string_value(obj: list, entity_data_dict: dict):
+        """ Adds string value to definition list
+
+        Args:
+            obj (list): containing the logical adis files and their contents as builtin types without string values
+            entity_data_dict (dict): dictionary of mapping values for \
+             entity numbers (e.g. {"0080004": "Betriebsnummer"})
+
+        Returns:
+             list: containing the logical adis files and their contents as builtin types with string values
+
+        """
+        for defs in obj[0]:
```

Comment:
it could be possible that there are multiple logical adis files (`obj`). so I think it is needed to iterate over those files as well